chainer/datasets/mnist.py

```diff
@@ -0,0 +1,107 @@
+import gzip
+import os
+import struct
+
+import numpy
+import six
+
+from chainer.dataset import download
+from chainer.datasets import tuple_dataset
+
+
+def get_mnist(withlabel=True, ndim=1, scale=1.):
+    """Gets the MNIST dataset.
+
+    `MNIST <http://yann.lecun.com/exdb/mnist/>`_ is a set of hand-written
+    digits represented by grey-scale 28x28 images. Each pixel is scaled to
```

Comment:
The sentence starts with "Each pixel ..." is not appropriate to be written here because it is not the description of MNIST but the operation which we do in this function on MNIST dataset.